### DISCUSSION TITLE
aria2下载文件保存位置不对。

### DISCUSSION THREAD

**Discussion by lingPoint** [AlistGo/alist#2151] [Q&A]
请问是配置问题还是BUG。
![image](https://user-images.githubusercontent.com/44696270/198825979-26625bc9-d566-4a95-9f44-7ae332fdcd8e.png)
![image](https://user-images.githubusercontent.com/44696270/198826069-2ba9a6e5-b6ec-43e1-91ba-26ac03ee9984.png)

**Comment by zx2587232768** [comment]
alist只是调用aria2进行下载，下载的地址实际有alist制定，下载的位置就在你alist所在目录/data/temp/ 这里
